README.md

```diff
@@ -1,6 +1,55 @@
-# curriculum-cr-training-mobile-menu
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# CR TRAINING: MOBILE MENU
 
 
 This repo host an initial code for the tasks performed during the Code Reviewers training.
 
 Application is based on this [Figma design](https://www.figma.com/file/t3EJUCAEViw3QasuJLPLVT/Microverse-Student-Potfolio-Templates-Main?node-id=1%3A1471).
+
+
+## Built With
+
+- HTML
+- CSS
+- JAVASCRIPT
+- VS CODE
+
+
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+
+### Setup
+- Clone the repository
+- cd into the cloned repo directory using the command line
+- Open it in your preferred code editor
```

Comment:
- [ ] **OPTIONAL** Please be more specified when it comes to the instructions for cloning and setting up the repo locally because there might be non-technicals who are viewing your app.
EX:
to clone the repo run this command line in your terminal `git clone <repo link>`
and so on...